msal_extensions/persistence.py

```diff
@@ -119,9 +119,11 @@ def save(self, content):
             handle.write(self._dp_agent.protect(content))
 
     def load(self):
-        # type: () -> str
+        """Return decrypted content, or None if input is empty."""
+        # type: () -> Optional[str]
         with open(self._location, 'rb') as handle:
-            return self._dp_agent.unprotect(handle.read())
+            data = handle.read()
+            return self._dp_agent.unprotect(data) if data else None
```

Comment:
We should have been doing this, but we did not. Since we now have an explicit `data` anyway, let's do it the right way then.
```suggestion
        return self._dp_agent.unprotect(data) if data else None
```